web/src/pages/calendar.tsx

```diff
@@ -22,7 +24,25 @@ export const getStaticProps = (async () => {
 const EventCalendarPage = ({
   events,
 }: InferGetStaticPropsType<typeof getStaticProps>) => {
-  return <Calendar events={events} />;
+  const [viewType, setViewType] = useState<'calendar' | 'agenda'>('calendar');
+  const [isMobile, setMobile] = useState(false);
+  const {width} = useWindowSize();
+
+  useEffect(()=>{
+    if (width && width < 700) {
+      setMobile(true);
+      setViewType('agenda');
+    } else {
+      setMobile(false);
+    }
```

Comment:
I noticed that it won't toggle back to a calendar if you resize it wider, which seems desireable? What do you think?
```suggestion
    if (width && width < 700) {
      setMobile(true);
      setViewType('agenda');
    } else {
      setMobile(false);
      setViewType('calendar');
    }
```